api-reference/_hidden/dxDataGridColumn/calculateGroupValue.md

```diff
@@ -4,16 +4,170 @@ type: String | function(rowData)
 ---
 ---
 ##### shortDescription
-Specifies a field name or a function that returns a field name or a value to be used for grouping column cells.
+A field name or a function that returns a field name or value by which to group cells. This value is also used to sort **DataGrid**'s groups.
```

Comment:
"A field name" "that returns a field name"?  Can you explain?

Reply:
calculateGroupValue specifies a field name OR a function. This function should return a field name or value by which to group cells.
If this sentence is ambiguous, here is another variant: "A field name by which to group column cells or a function that returns a field name or a calculated value to perform this operation."
Is it good?

Reply:
A field name by which to group column cells or a function that returns a field name or a calculated value for this operation?